src/main/cpp/t04_students.cpp

```diff
@@ -32,7 +32,54 @@
 
 using namespace std;
 
-int t04_students() {
-    
+struct student{
+  string name;
+  string family;
+  int m;
+  int f;
+  int i;
+};
+
+double mejdinno(student s)
+{
+  return (double)(s.m+s.f+s.i)/3;
 }
 
+void sort(student arr[], int n)
+{
+  bool swapped;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'swapped' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4594315)